### PR TITLE
Merge bitcoin#9963: util: Properly handle errors during log message formatting

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Decided to break this off the main set since it should have been in like pr2 and I missed it. I could cherry-pick this into pr10 if wanted

b651270 util: Throw tinyformat::format_error on formatting error (Wladimir J. van der Laan)
3b092bd util: Properly handle errors during log message formatting (Wladimir J. van der Laan)

Tree-SHA512: 85e3b7afec2255fc88034187f1abd6060e9421de17ed4e3d918416f393429a99cc2c974b362099aaaff6970549df47664bea4c857c4e46acc0789663201dc541

### CODE REVIEW

**Comment by UdjinM6** [comment]
```
llmq/quorums.cpp:215:21: note: in expansion of macro 'LogPrint'
                     LogPrint("llmq", debugMsg);
```
https://travis-ci.org/dashpay/dash/jobs/531315434#L2159

File: src/llmq/quorums.cpp [hunk 1]
```diff
@@ -212,7 +212,7 @@ void CQuorumManager::EnsureQuorumConnections(Consensus::LLMQType llmqType, const
                             debugMsg += strprintf("  %s (%s)\n", c.ToString(), dmn->pdmnState->addr.ToString(false));
                         }
                     }
-                    LogPrint("llmq", debugMsg);
```
Comment (PastaPastaPasta) [inline src/llmq/quorums.cpp]: thoughts on "casting" this way @UdjinM6 ?

Comment (UdjinM6) [reply src/llmq/quorums.cpp]: Sounds good imo

Comment (UdjinM6) [reply src/llmq/quorums.cpp]: There is another one https://github.com/dashpay/dash/blob/develop/src/llmq/quorums_dkgsessionhandler.cpp#L492

Comment (PastaPastaPasta) [reply src/llmq/quorums.cpp]: resolved and force pushed

**Pull request merged**